scripts/build-number.sh

```diff
@@ -0,0 +1,7 @@
+version=$(git rev-parse HEAD | cut -c-8)
+
+if [ -z "$CI" ]; then
+  version="$version-dev"
+fi
+
+echo $version
```

Comment:
Probably nothing, but echo might output a trailing new line there. Just to be safe `echo -n` will not output a new line. I don't know how that's handled from rust and js though, so might be nothing.

Reply:
good call! Rust does the right thing but js needed `trim`. I just tried `echo -n` and it's working when called directly not from the script, but I just realized this is what `printf` is for! 😄